components/experimental/src/dimension/units/format.rs

```diff
@@ -53,35 +53,47 @@ impl<'l> Writeable for FormattedUnit<'l> {
     }
 }
 
+impl core::fmt::Display for FormattedUnit<'_> {
```

Comment:
This is required by `assert_writeable_eq!` macro, or  I need to implement `core::fmt::ToString` trait

Reply:
use https://docs.rs/writeable/latest/writeable/macro.impl_display_with_writeable.html